documentation/modules/exploit/multi/http/wp_plugin_sp_project_document_rce.md

```diff
@@ -0,0 +1,76 @@
+## Vulnerable Application
+
+### Description
+
+This module allows an attacker with a privileged Wordpress account to launch a reverse shell
+due to an arbitrary file upload vulnerability in Wordpress plugin SP Project & Document < 4.22.
+This is due to an incorrect check of the uploaded file extension which should be of SGBP type.
```

Comment:
Looks like this line is Wordpress Backup Guard copypasta

Reply:
Yes well seen !